spec/spec_helper.rb

```diff
@@ -0,0 +1,8 @@
+RSpec.configure do |config|
+  config.expect_with :rspec do |expectations|
+    expectations.include_chain_clauses_in_custom_matcher_descriptions = true
+    end
```

Comment:
`end` at 4, 4 is not aligned with `config.expect_with :rspec do |expectations|` at 2, 2